jumanji/environments/logic/minesweeper/utils_test.py

```diff
@@ -55,11 +55,11 @@ def test_explored_mine(
     expected_explored_mine_result: bool,
 ) -> None:
     """Test whether mines are being explored"""
-    action_height, action_width = action
+    action_row, action_col = action
     assert (
         explored_mine(
             manual_start_state,
-            jnp.array([action_height, action_width], dtype=jnp.int32),
+            jnp.array([action_row, action_col], dtype=jnp.int32),
```

Comment:
That's actually your action, isn't it?
```suggestion
            action,
```